web/js/map-OpenLayers.js

```diff
@@ -710,48 +697,29 @@ OpenLayers.Control.Click = OpenLayers.Class(OpenLayers.Control, {
         // Store pin location in form fields, and check coverage of point
         fixmystreet_update_pin(lonlat);
 
-        // Already did this first time map was clicked, so no need to do it again.
-        if (fixmystreet.page == 'new') {
-            return;
+        // If there are notes to be displayed, add the .with-notes class
+        // to make the sidebar wider.
+        if($('#report-a-problem-sidebar').length){
+          $('#map_box, #map_sidebar').addClass('with-notes');
```

Comment:
Yes: `[% TRY %][% sidebar_html = PROCESS 'report/new/sidebar.html' %][% CATCH file %][% END %]` should do it, the you can use that in an `IF` and `[% sidebar_html %]` to print it out.